pkg/server/api.go

```diff
@@ -29,11 +29,11 @@ func (gs *Server) Evaluate(ctx context.Context, proposedTransaction *pb.Proposed
 	endorsers := gs.registry.GetEndorsers(channel, chaincodeID)
 	fmt.Printf("Evaluate: #endorsers=%d\n", len(endorsers))
 	if len(endorsers) == 0 {
-		return nil, fmt.Errorf("no endorsing peers found for channel: %s", channel)
+		return nil, fmt.Errorf("No endorsing peers found for channel: %s", proposedTransaction.ChannelId)
```

Comment:
Error strings should not start with capital letters according to:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings
This shows as warnings in my editor.
Several occurrences of this.